### PR TITLE
Reduced flakiness of HPA e2e tests.

### PR DESCRIPTION
**Pull request by jszczepkowski**
[Analytics]()

Reduced flakiness of HPA e2e tests.

### CODE REVIEW

**Comment by jszczepkowski** [comment]
CC @fgrzadkowski

File: test/e2e/horizontal_pod_autoscaling.go [hunk 1]
```diff
@@ -136,7 +136,11 @@ func (scaleTest *HPAScaleTest) run(name, kind string, rc *ResourceConsumer, f *f
...
 
-func scaleUp(name, kind string, rc *ResourceConsumer, f *framework.Framework) {
+func scaleUp(name, kind string, checkStability bool, rc *ResourceConsumer, f *framework.Framework) {
+	stasis := 0 * time.Minute
+	if checkStability {
+		stasis = 10 * time.Minute
+	}
 	scaleTest := &HPAScaleTest{
 		initPods:                    1,
 		totalInitialCPUUsage:        250,
```
Comment (piosz) [inline test/e2e/horizontal_pod_autoscaling.go]: Should we tune also this value?

**Comment by piosz** [comment]
LGTM, just a question.

File: test/e2e/horizontal_pod_autoscaling.go (hunk 1 above)
Comment (jszczepkowski) [inline test/e2e/horizontal_pod_autoscaling.go]: This value is exactly in the middle 250 = 2.5 * 100

**Comment by piosz** [comment]
@k8s-bot unit test this issue: #27068

**Comment by piosz** [comment]
This should help for flaky tests so bumping the priority.

**Pull request merged**